examples/performance_analysis/aep_performance/case_a_aep_3x3_simple.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2020 NREL
+
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+# See https://floris.readthedocs.io for documentation
+
+# In the second case, simply time calls to calculate_wake
+
+import time
+import pickle
+import itertools
+
+import numpy as np
+import matplotlib.pyplot as plt
+
+import floris.tools as wfct
+
+
+# PARAMETERS
+recompute_baseline = False
+show_layout = False
+turn_off_gch = True
+
+# Fixed parameters
+N_row = 3
+
+
+# Initialize the FLORIS interface fi
+# For basic usage, the florice interface provides a simplified interface to
+# the underlying classes
+fi = wfct.floris_interface.FlorisInterface("../../example_input.json")
+# fi = wfct.floris_interface.FlorisInterface("../../example_input_4_point.json")
+
+if turn_off_gch:
+    fi.set_gch(False)
+
+# Set to a 5 turbine case
+D = fi.floris.farm.turbines[0].rotor_diameter
+spc = 5
+layout_x = []
+layout_y = []
+for i in range(N_row):
+    for k in range(N_row):
+        layout_x.append(i * spc * D)
+        layout_y.append(k * spc * D)
```

Comment:
Could we use something more like the below to remove the for loops? I was more or less thinking in terms of simple speedups/cleanups. Feel free to ignore if the original feels more clear or anything like that though.

```python
from itertools import chain, product
layout_x = list(chain.from_iterable(([[i * spc * D] * N_row for i in range(N_row)])))
layout_y = [k * spc * D for _, k in product(range(N_row), range(N_row))]
```